src/modules/land_detector/VtolLandDetector.cpp

```diff
@@ -78,9 +78,7 @@ bool VtolLandDetector::_get_ground_contact_state()
 
 bool VtolLandDetector::_get_maybe_landed_state()
 {
-
-	// TODO
-	return false;
+	return MulticopterLandDetector::_get_maybe_landed_state();
```

Comment:
Doesnt work

src/modules/land_detector/libmodules__land_detector.a(VtolLandDetector.cpp.o):(.rodata._ZTVN13land_detector16VtolLandDetectorE[_ZTVN13land_detector16VtolLandDetectorE]+0x58): undefined reference to `land_detector::VtolLandDetector::_get_maybe_landed_state()'
src/modules/land_detector/libmodules__land_detector.a(VtolLandDetector.cpp.o):(.rodata._ZTVN13land_detector16VtolLandDetectorE[_ZTVN13land_detector16VtolLandDetectorE]+0x60): undefined reference to `land_detector::VtolLandDetector::_get_ground_contact_state()'
collect2: error: ld returned 1 exit status